options/locale/locale_en-US.ini

```diff
@@ -1957,6 +1957,9 @@ topic.done = Done
 topic.count_prompt = You can not select more than 25 topics
 topic.format_prompt = Topics must start with a letter or number, can include dashes ('-') and can be up to 35 characters long.
 
+packages = packages
+packages.desc = permission unit for third part packages
```

Comment:
I don't mean to be rude but it should be third *party*.
feel free to tell me if spell-checking is not welcome. 🙈